pipelines/skylab/smartseq2_single_nucleus/mouse_paired_example.json

```diff
@@ -0,0 +1,11 @@
+{
+  "SmartSeq2SingleNucleus.genome_ref_fasta": "gs://hca-dcp-sc-pipelines-test-data/referenceData/references/modified_GRCm38.primary_assembly.genome.fa",
+  "SmartSeq2SingleNucleus.tar_star_reference": "gs://hca-dcp-sc-pipelines-test-data/referenceData/references/modified_star_v2.7.8a_primary_gencode_mouse_vM23.tar",
+  "SmartSeq2SingleNucleus.annotations_gtf": "gs://hca-dcp-sc-pipelines-test-data/referenceData/references/introns_modified_gencode.vM23.primary_assembly.annotation.gtf",
+  "SmartSeq2SingleNucleus.adapter_list": "gs://hca-dcp-sc-pipelines-test-data/adapters/Illumina_adapters_list.fa",
+  "SmartSeq2SingleNucleus.fastq1":"gs://hca-dcp-mint-test-data/datasets/alleninstitute/SM-GE644_S110_E1-50_GGAGCTAC-CTAAGCCT_R1.fastq.gz",
```

Comment:
Did we ask allen about these being public/for test data?

Reply:
Yes, these files are in NeMO and are public.